libs/platform/src/lib/components/upload-collection/upload-collection/upload-collection.component.html

```diff
@@ -350,28 +350,29 @@ <h3 [innerText]="noDataText"></h3>
     <fdp-menu-item
         *ngFor="let value of _itemsPerPageOptions"
         (itemSelect)="_itemsPerPageChange(value.label)"
+        [attr.title]="'Results per page: ' + value.label"
         [innerText]="value.label"
     ></fdp-menu-item>
 </fdp-menu>
 
 <fdp-menu #menuActions xPosition="before" contentDensity="compact">
     <fdp-menu-item (itemSelect)="_moveToClicked()">
-        <fd-icon class="fdp-upload-collection__fd-icon" glyph="folder-blank"></fd-icon>
-        <span i18n="platformUploadCollection.Btn.MoveTo|Menu Action">Move to</span>
+        <fd-icon class="fdp-upload-collection__fd-icon" glyph="folder-blank" i18n-title="platformUploadCollection.Icon.MoveTo|Menu Action" title="Move to icon"></fd-icon>
+        <span i18n="platformUploadCollection.Btn.MoveTo|Menu Action" i18n-title="platformUploadCollection.Btn.MoveTo.Title|Menu Action" title="Move to">Move to</span>
```

Comment:
Do you really need `title` for the text elements?

Reply:
fixed